### PR TITLE
css-cascade complete

### PR DESCRIPTION
**Pull request by luiderek**
image
image
image

### CODE REVIEW

File: css-cascade/specificity.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+* {
```
Comment (Zhiwutian) [inline css-cascade/specificity.css:1]: These selectors should be in the reverse order to demonstrate that source order will be overridden by a more specific selector

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed

File: css-cascade/specificity.css (hunk 1 above)
Comment (luiderek) [reply css-cascade/specificity.css]: gotcha, redid that section of the exercise

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**